Scripts/CheckBinfmtNotInstall.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/sh
+result=0
+if command -v update-binfmts>/dev/null; then
+  update-binfmts --find "$1" 1>&- 2>&-
+  if [ $? -eq 0 ]
+  then
+    # If we found the binfmt_misc file passed in then error
+    result=1
+  fi
+fi
+
+if [ $result -eq 0 ]
+then
+  if [ -f "$1" ]; then
+    # If the binfmt_misc file exists then error
+    result=1
+  fi
+fi
+
+if [ $result -eq 1 ]
+then
+  echo "==============================================================="
+  echo "$1 binfmt file is installed!"
+  echo "This conflicts with FEX-Emu's binfmt_misc!"
+  echo "This will cause issues when running FEX-Emu through binfmt_misc"
+  echo "Not installing until you uninstall this binfmt_misc file!"
```

Comment:
For people who are following install instructions but aren't familiar with binfmts, can we provide more helpful instructions on how to uninstall the existing file?

Reply:
We don't know how a user will have installed these binfmt_misc files. So concise instructions isn't really viable.
We can provide a direct method of uninstalling it but package updates for qemu-user-static would break it again.
Not sure if Box has packaged versions.

Reply:
Hmm right. I guess it is better to leave it unspecified then indeed to encourage "only proceed if you know what you're doing" behavior here.